src/Aquifer.Data/Entities/BibleBookContentEntity.cs

```diff
@@ -22,4 +27,38 @@ public class BibleBookContentEntity : IHasUpdatedTimestamp
 
     [SqlDefaultValue("getutcdate()")]
     public DateTime Updated { get; set; } = DateTime.UtcNow;
+
+    /// <summary>
+    /// POCO for deserializing the AudioUrls column.
+    /// </summary>
+    public sealed class AudioUrlsData
+    {
+        public required IReadOnlyList<Chapter> Chapters { get; set; }
+
+        public sealed class Chapter
+        {
+            /// <summary>
+            /// The chapter number is a JSON string type but is currently only populated with integer values (this might change in the future).
+            /// </summary>
+            public required string Number { get; set; }
+            public required AudioUrl? Webm { get; set; }
+            public required AudioUrl? Mp3 { get; set; }
+            public required IReadOnlyList<AudioTimestamp> AudioTimestamps { get; set; }
+
+            public sealed class AudioUrl
+            {
+                public required string Url { get; set; }
+                public required int Size { get; set; }
+            }
+            public sealed class AudioTimestamp
+            {
+                /// <summary>
+                /// The verse number is a JSON string type but is currently only populated with integer values (this might change in the future).
+                /// </summary>
+                public required string VerseNumber { get; set; }
+                public required decimal Start { get; set; }
```

Comment:
Nit: Do we need decimal here as opposed to float? I think decimal has more overhead, and don't think we need to worry about the precision here since it's just gonna get passed as is anyway.

Reply:
Yes, decimal has more overhead, though that overhead should be insignificant for only a few fields.  I considered using `float` and opted for the precision of decimal which should guarantee the data from the JSON string is represented in .net precisely without strange rounding errors.  See https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/floating-point-numeric-types:

> The decimal type is appropriate when the required degree of precision is determined by the number of digits to the right of the decimal point. Such numbers are commonly used in financial applications, for currency amounts (for example, $1.00), interest rates (for example, 2.625%), and so forth. Even numbers that are precise to only one decimal digit are handled more accurately by the decimal type: 0.1, for example, can be exactly represented by a decimal instance, while there's no double or float instance that exactly represents 0.1. Because of this difference in numeric types, unexpected rounding errors can occur in arithmetic calculations when you use double or float for decimal data. You can use double instead of decimal when optimizing performance is more important than ensuring accuracy. However, any difference in performance would go unnoticed by all but the most calculation-intensive applications. Another possible reason to avoid decimal is to minimize storage requirements. For example, [ML.NET](https://learn.microsoft.com/en-us/dotnet/machine-learning/how-does-mldotnet-work) uses float because the difference between 4 bytes and 16 bytes adds up for very large data sets. For more information, see [System.Decimal](https://learn.microsoft.com/en-us/dotnet/api/system.decimal).

Do rounding errors on partial seconds within an occasional timestamp matter here?  Probably not that much.  But the overhead to remove them by using `decimal` seems acceptable.

Reply:
I guess I think about it more around if you're doing operations with the numbers, and these are just read-only, so it wouldn't matter. But either way.